.github/workflows/ci-vulnerability-scans.yml

```diff
@@ -33,8 +33,16 @@ jobs:
         uses: hadolint/hadolint-action@v3.1.0
         with:
           dockerfile: app/Dockerfile
-          format: json
+          format: tty
           failure-threshold: warning
+          output-file: hadolint-results.txt
+
+      - name: Save output to workflow summary
+        run: |
```

Comment:
just wondering, do we need the `if: always()` like we do with some of the other actions?

Reply:
Good catch, in next commit